pkg/test/e2e/ccm-migration/ccm_migration_suite_test.go

```diff
@@ -26,39 +26,50 @@ import (
 	"github.com/onsi/gomega"
 
 	"k8c.io/kubermatic/v2/pkg/semver"
+	"k8c.io/kubermatic/v2/pkg/test/e2e/ccm-migration/providers"
 	e2eutils "k8c.io/kubermatic/v2/pkg/test/e2e/utils"
 )
 
 // Options holds the e2e test options.
 type testOptions struct {
 	skipCleanup       bool
 	debugLog          bool
-	datacenter        string
-	userClusterName   string
 	kubernetesVersion semver.Semver
 
-	osCredentials credentials
+	vsphereSeedDatacenter string
+	osSeedDatacenter      string
+
+	osCredentials      providers.OpenstackCredentialsType
+	vSphereCredentials providers.VsphereCredentialsType
 }
 
 var options = testOptions{
-	kubernetesVersion: *semver.NewSemverOrDie("v1.20.14"),
+	kubernetesVersion: *semver.NewSemverOrDie("v1.21.7"),
```

Comment:
Is it possible that we provide the Kubernetes version as an environment variable? We already do that for other jobs.

https://github.com/kubermatic/kubermatic/blob/e2f72a7beb8a74827af0f050e458a7f88a49cd4b/.prow.yaml#L410-L411